config/manager/kustomization.yaml

```diff
@@ -13,4 +13,4 @@ kind: Kustomization
 images:
 - name: perconalab/percona-server-mysql-operator
   newName: perconalab/percona-server-mysql-operator
-  newTag: main
+  newTag: 2.0.0-alpha
```

Comment:
Why do we need it in main branch ?

Reply:
I'll update versions to main again after we merge and tag the release branch